### DISCUSSION TITLE
app/HiddifyCli-guide/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4071] [Content Help کمک به اصلاح محتوا]
# app/HiddifyCli-guide/

The proxy manager platform.

https://hiddify.com/app/HiddifyCli-guide/?h=command+line

**Comment by amireza007** [comment]
How **ON EARTH** should I set the "log level" in the config file to "none"? No matter what I write in the config.json file, the `HiddifyCli` would change it to the following in the json file: 
` "log": { 
    "level": "warn", 
    "timestamp": true
}`

**Comment by mfat** [comment]
How to make config file?

**Reply by amireza007** [reply]
اگر برنامه ی هیدیفای رو نصب و توش کانفیگی دارید، توی ویندوز میتونید کانفیگ رو بردارید و از اون استفاده کنید.
کانفیگ جیسانش اینجاست: یه پنجره اکسپلورر باز کنید و این آدرسو بنویسین
```
%APPDATA%\Hiddify
``` 
`or C:\Users\USERNAME\AppData\Roaming\Hiddify`

بعد یه پوشه دیگه هیدیفایم احتمالا باشه، برید داخلش و از فایل `current-config.json` استفاده کنید.

**Comment by mfat** [comment]
ممنون از اپ اندروید اکسترکت کردم.

**Reply by amireza007** [reply]
احسنت 👌👍 کلن داکیومنتیشن cli اش لِه عه!! 😩
